### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Profsain**
In this project, I built a weather app using openweathermap API.
- Added app project setup file
- Added UI component
- Fetched weather data from API
- Display weather data in the UI
- Implemented search weather by city name functionality
- Update the Readme file
-
screencapture-localhost-3000-2022-08-04-12_11_29

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,47 @@
-# Getting Started with Create React App
+# City-Weather App
+A react-redux web app that help users to search for weather condition of any city, view location & date, view wind status & wind direction, view humidity, air pressure, view weather condition of the next 5 days, and convert from degree Celsius to Fahrenheit or vice visa
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+### Live Demo
```
Comment (Whoistolu) [inline README.md:4]: - [ ] [REQUIRED] Kindly ensure that this project is been deployed using Heroku, Netlify, or GitHub Pages as stated in the project requirement.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,47 @@
-# Getting Started with Create React App
+# City-Weather App
```
Comment (Whoistolu) [inline README.md:1]: - [ ] [REQUIRED] Kindly ensure you make a video presentation of this project with a 5-minute maximum length. Also, ensure the video has a clear presentation of the project with a highlight of a piece of code you are most proud of. This is a core requirement.

File: README.md [hunk 3]
```diff
@@ -1,70 +1,47 @@
...
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+Acknowledgments
+Hat tip to anyone whose code was used
+Microverse Community
```
Comment (Whoistolu) [inline README.md]: - [ ] [REQUIRED] In the Acknowledgment section, kindly ensure that you give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) as stated in the project requirement.

File: README.md (hunk 1 above)
Comment (Whoistolu) [inline README.md:4]: - [ ] [REQUIRED] Kindly ensure that you deploy this application using Heroku, Netlify, or GitHub Pages as stated in the project requirement.

File: src/components/SingleCard.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        {temp}
+        ℃
+      </p>
+      <p>
+        {humidity}
+        %
+      </p>
+    </div>
+  </div>
+);
```
Comment (Whoistolu) [inline src/components/SingleCard.js:26]: - [ ] [REQUIRED] You have put a great effort into this project. I can see that whenever search for a state of the country Nigeria, it updates the state and also fetches my current location which is very good. I also discovered that your details are been displayed on that same page. However, note that this project requires that you put the state of the country on the homepage and display its details on the details page. See again the image of the template provided for this project and as I have explained to you on our last call.
### Template Provided
Screenshot from 2022-08-04 18-36-31

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Profsain 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 5]
```diff
@@ -1,70 +1,50 @@
-# Getting Started with Create React App
+# City-Weather App
+A react-redux web app that help users to search for weather condition of any city, view location & date, view wind status & wind direction, view humidity, air pressure, view weather condition of the next 5 days, and convert from degree Celsius to Fahrenheit or vice visa
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+### Live Demo
+[City-Weather App:](https://joyful-arithmetic-e662f9.netlify.app/)
 
-## Available Scripts
+[Video Demo:](https://www.loom.com/share/0046e2bc7a3b4553b649f7dcd167f784)
```
Comment (AmaduKamara) [inline README.md]: Good job on making a video presentation for your project as required by the project and the previous reviewer, but you still have a few more areas to mention in the video that you left out and it is required you mention them in the video.

- [ ] Kindly mention a code section that you are proud of implementing. The idea here is for you to show a basic code section in the project that you feel really proud working on. Please redo the video, and mention that part 👍

File: README.md [hunk 6]
```diff
@@ -1,70 +1,50 @@
...
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+Acknowledgments
+Hat tip to anyone whose code was used
+Microverse Community for their support
+[www.devchallenge.io](https://devchallenges.io/) for the design inspiration
```
Comment (AmaduKamara) [inline README.md]: - [ ] Just as the previous reviewer asked, please give credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required in the project description. Microverse used that design to give you the inspiration to build something similar to it and it is required that you give credit to them.

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @Profsain,

Good job so far on working on the details page and a few other changes asked by the previous reviewer.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
You did well on the following: 
✅ Used Gitflow correctly
✅ All linter checks passed
✅ Good Descriptive PR
✅ Good commit messages
✅ App is hosted on Netlify 👍 

## Required Changes ♻️
_Check the comments under the review._

The Previous reviewer did not mention this one and it's required for this project 

- [ ] Kindly write tests for this project as it's required you write unit tests for most of your components.
- 
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by julie-ify — APPROVED** [review]
Hi @Profsain,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**